src/dom/html/WindowRe.re

```diff
@@ -95,6 +95,28 @@ module Impl (T: { type t; }) => {
 
 
   external setOnLoad : t_window => (unit => unit) => unit = "onload" [@@bs.set]; /* use addEventListener instead? */
+
+  module Image = {
+    type t;
+    external make : unit => t = "Image" [@@bs.new];
+    external src : t => string => unit = "" [@@bs.set];
+    external getSrc : t => string = "src" [@@bs.get];
+    external onload : t => (unit => unit) => unit = "" [@@bs.set];
+    external naturalHeight : t => string = "" [@@bs.get];
+    external naturalWidth : t => string = "" [@@bs.get];
+  };
+  
+  module File = {
```

Comment:
This is part of the File API, not the DOM, and so should be in `src/file/FileRe.re` or something.

Also, while I understand that these implementations are incomplete, how would you even acquire a `File.t` instance?

Reply:
This is inferred in our project when using a file input where we send the type into a function that accepts a File.t

```ocaml
let file = List.hd (ReactDOMRe.domElementToObj (ReactEventRe.Form.target event))##files;
```